Objects/Converters/ConverterCSI/ConverterCSIShared/ConverterCSIUtils.cs

```diff
@@ -19,9 +19,10 @@ public partial class ConverterCSI
     readonly string Send2DResults = "send2DResults";
 
     private string _modelUnits;
+
     public string ModelUnits()
     {
-      if (_modelUnits != null) 
+      if (_modelUnits != null)
         return _modelUnits;
```

Comment:
I'd even go with brackets here tbh, for single line - do we at least have line space after?

Reply:
CSharpier doesn't remove brackets or any other potentially dangerous changes, just enforces consistent spacing, line-breaks, etc...  

We can also enforce higher level style's such as disallowing single line if statements without brackets with the current setup, but we'd use dotnet format or resharper for that (resharper is waaay better, but not fast on CLI)

Devs already get info messages on their code based on the current preference (which i think is no brackets for single-line statements), but this is not really enforced.